### PR TITLE
Enabling spinning a local dev environment with: make, docker-compose …

### PR DESCRIPTION
**Pull request by lioramilbaum**
…& docker

### CODE REVIEW

**Comment by lioramilbaum** [comment]
@mprahl The image is built but the end point is not accessible on run-time. Can you please take a look?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM registry.redhat.io/ubi8/nodejs-10:latest
+
+ADD . .
+
+RUN npm install
+
+EXPOSE 4200
+
+CMD ng serve --open
```
Comment (mprahl) [inline Dockerfile:9]: Try this instead:
```suggestion
CMD ng serve --host 0.0.0.0
```

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: '3.9'
```
Comment (mprahl) [inline docker-compose.yml:1]: Could we please use 3.3 for greater compatibility? That's what my client requires (Fedora 32).

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+services:
+  frontend:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: frontend
+    ports:
+      - 4200:4200
+    expose:
```
Comment (mprahl) [inline docker-compose.yml:10]: I believe this is redundant since it's in the Dockerfile already.

**Review by mprahl — CHANGES_REQUESTED** [review]
Thanks for the contribution. Just a few small changes.

File: Dockerfile (hunk 1 above)
Comment (lioramilbaum) [reply Dockerfile]: Thanks. That worked.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,9 @@
+FROM registry.redhat.io/ubi8/nodejs-10:latest
+
+ADD . .
+
+RUN npm install
+
+EXPOSE 4200
+
+CMD ng serve --host 0.0.0.0
```
Comment (mprahl) [inline Dockerfile:9]: Could you add a newline here please?

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,9 @@
+FROM registry.redhat.io/ubi8/nodejs-10:latest
```
Comment (mprahl) [inline Dockerfile:1]: Do you think we could rename this Dockerfile to indicate that it's for development only and not for production use? For instance, we can call it `Dockerfile-dev` or something.

**Review by mprahl — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -4,13 +4,17 @@ Visualizes the story an artifact takes in the Red Hat build to release pipeline.
 
 ## Development
 
-To setup a development environment, make sure you have `npm` installed.
+### Prerequests
```
Comment (mprahl) [inline README.md:7]:
```suggestion
### Prerequisites
```

**Review by mprahl — CHANGES_REQUESTED** [review]
Just one typo to fix. Sorry I missed this in the last review.

**Review by mprahl — APPROVED** [review]

**Pull request merged**